### PR TITLE
Log the time it takes to sync the file on disk.

### PR DESCRIPTION
**Pull request by DiegoDiverio**
Fixes https://github.com/channable/icepeak/issues/91. Didn't add the timestamp because we can see it with other tools anyway.

### CODE REVIEW

File: server/src/Persistence.hs [hunk 1]
```diff
@@ -155,7 +156,12 @@ loadFromBackend backend config = runExceptT $ do
   return val
 
 syncToBackend :: StorageBackend -> PersistentValue -> IO ()
-syncToBackend File pv = syncFile pv
+syncToBackend File pv = do
+    start <- Clock.getTime Clock.Monotonic
+    syncFile pv
+    end <- Clock.getTime Clock.Monotonic
+    let time = Clock.toNanoSecs $ (Clock.diffTimeSpec end start) `div` 1000000
```
Comment (fatho) [inline server/src/Persistence.hs:163]: In the end, it probably doesn't matter, but isn't the order of operations conceptually wrong here?

The `Clock.toNanoSecs` spans the whole division, so despite its name, it returns milliseconds.

Just removing the $ should result in the same output in a less confusing way:  ``Clock.toNanoSecs (Clock.diffTimeSpec end start) `div` 1000000``

**Review by fatho — COMMENTED** [review]
Thanks, Just a small thing:

File: server/src/Persistence.hs (hunk 1 above)
Comment (DiegoDiverio) [reply server/src/Persistence.hs:163]: Yep, it's conceptually better without the `$`. I'll remove that!

**Review by fatho — APPROVED** [review]
:shipit:

**Comment by DiegoDiverio** [comment]
@OpsBotPrime merge

**Comment by OpsBotPrime** [comment]
Rebased as 16aebfc9095da2239ec5912b85b0e3a1fc09a26d, waiting for CI …

**Comment by OpsBotPrime** [comment]
[CI job :yellow_circle:](https://channable.semaphoreci.com/workflows/2cd25e54-577c-48ee-90f2-7423218aab9f?pipeline_id=fbb879a3-fbce-4901-bfb9-78a7cb2c2d07) started.

**Pull request closed without merging**